<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="mominul-10662" ignored="0" name="Test1" tests="1" failures="1" timestamp="2020-09-27T01:06:50 BDT" time="10.868" errors="0">
  <testcase name="testOneDemo" time="9.948" classname="tests.OneTest">
    <failure type="java.lang.AssertionError" message="expected [Your] but found [Your Store]">
      <![CDATA[java.lang.AssertionError: expected [Your] but found [Your Store]
at tests.OneTest.testOneDemo(OneTest.java:20)
at java.util.ArrayList.forEach(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- testOneDemo -->
</testsuite> <!-- Test1 -->
